server/http_handler.go

```diff
@@ -539,9 +568,8 @@ type IndexRegions struct {
 // RegionDetail is the response data for get region by ID
 // it includes indices and records detail in current region.
 type RegionDetail struct {
+	RangeDetail
```

Comment:
```suggestion
	RangeDetail `json:",inline"`
```
To keep back compatible.